docs/searchqueryset_api.rst

```diff
@@ -323,6 +323,23 @@ Example::
     # Count document hits for each author within the index.
     SearchQuerySet().filter(content='foo').facet('author')
 
+``interval_facet``
+~~~~~~~~~~~~~~
+
+.. method:: SearchQuerySet.interval_facet(self, field, interval)
+
+Adds interval faceting to a query for the provided field. You provide the field
+(from one of the ``SearchIndex`` classes) you like to facet on and an array of
+``intervals``. This is currently supported only when using Solr (ver 4.10+)
```

Comment:
Reviewers: Should we drop support for Solr < v4.10 or should this code assert that the Solr version is >= 4.10?